src/pudl/package_data/eia860/column_maps/energy_storage_existing.csv

```diff
@@ -0,0 +1,38 @@
+﻿year_index,2016,2017,2018,2019,2020,2021,2022
+utility_id_eia,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id,utility_id
+utility_name_eia,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name,utility_name
+plant_id_eia,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code,plant_code
+plant_name_eia,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name,plant_name
+state,state,state,state,state,state,state,state
+county,county,county,county,county,county,county,county
+generator_id,generator_id,generator_id,generator_id,generator_id,generator_id,generator_id,generator_id
+operational_status_code,status,status,status,status,status,status,status
+storage_technology_type,technology,technology,technology,technology,technology,technology,technology
```

Comment:
hmmm yeah I was thinking about this too. I think it is storage specific because this is the storage table and you shouldn't expect to see any other type of technology.

I'm not sure what to do here mostly because we say that we want to standardize columns as much as possible, but we also have a history of adding prefixes to certain columns to differentiate them slightly. For example: `total_cost` and `operating_date` always have a prefix to specify what kind of total cost or operating date. 

I don't have a strong preference here, curious to hear your thoughts on this.

Reply:
iirc we added special prefixes to `total_cost` and `operating_date` to differential which entity they refer to. but this is actually a table with storage _generators_ w/ a bunch of storage-specific attributes and some more generic generator attributes. If we want to harvest any of these generator attributes they need to have the same name.